content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -167,14 +167,13 @@ option. Your cluster requirements may need a different configuration.
     {{< /caution >}}
 
 1.  Apply the CNI plugin of your choice:
-    [Follow these instructions](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network)
+    [Follow these instructions](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network) 
     to install the CNI provider. Make sure the configuration corresponds to the Pod CIDR specified in the kubeadm configuration file if applicable.
-
-    In this example we are using Weave Net:
-
-    ```sh
-    kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')"
-    ```
+  
+  {{<note>}}
+  You must pick your own CNI plugin and deploy it before you move on to next step.
```

Comment:
I'd write “you must pick a network plugin that suits your use case” or something like that. Readers might think that they should somehow actually own a network plugin.

Reply:
okay @sftim